css-four-column-layout/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="eng">
+
+<head>
+  <title>CSS Four Column</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="row-header">
+      <h2>How It Works</h2>
+    </div>
+    <div class="row-body-half">
```

Comment:
You are still using individually named `row` classes. Use one `row` class with just `display: flex` applied to it, and make additional classes for any other styling necessary on that element.

Reply:
This `row` name and the `row` name for the above row still need to be revised